README.md

```diff
@@ -23,9 +23,9 @@ _md5: 2dd2eeeda08ac8c15be8a9f2d01adbe8_
 Since you're an elite Android reverser, I'm sure you already have Smali and Baksmali on your path. If for some strange reason it's not already installed, this should get you started, but please examine it carefully before running:
 
 ```bash
-mkdir ~/bin || cd ~/bin
-curl --location -O https://bitbucket.org/JesusFreke/smali/downloads/smali-2.2.5.jar && mv smali-*.jar smali.jar
-curl --location -O https://bitbucket.org/JesusFreke/smali/downloads/baksmali-2.2.5.jar && mv baksmali-*.jar baksmali.jar
+mkdir ~/bin && cd ~/bin || cd ~/bin
+curl --location -O https://bitbucket.org/JesusFreke/smali/downloads/smali-2.2.6.jar && mv smali-*.jar smali.jar
+curl --location -O https://bitbucket.org/JesusFreke/smali/downloads/baksmali-2.2.6.jar && mv baksmali-*.jar baksmali.jar
```

Comment:
This is a good direction, but I can't accept this PR with the other change. Also, the newest version is 2.2.7 now.